cli/azd/internal/repository/app_init.go

```diff
@@ -235,107 +232,25 @@ func (i *Initializer) InitFromApp(
 	}
 
 	tracing.SetUsageAttributes(fields.AppInitLastStep.String("config"))
-
-	// Create the infra spec
-	var infraSpec *scaffold.InfraSpec
-	composeEnabled := i.features.IsEnabled(featureCompose)
-	if !composeEnabled { // backwards compatibility
-		spec, err := i.infraSpecFromDetect(ctx, detect)
-		if err != nil {
-			return err
-		}
-		infraSpec = &spec
-
-		// Prompt for environment before proceeding with generation
-		_, err = initializeEnv()
-		if err != nil {
-			return err
-		}
-	}
-
 	tracing.SetUsageAttributes(fields.AppInitLastStep.String("generate"))
 
 	title = "Generating " + output.WithHighLightFormat("./"+azdcontext.ProjectFileName)
 	i.console.ShowSpinner(ctx, title, input.Step)
-	err = i.genProjectFile(ctx, azdCtx, detect, composeEnabled)
+	err = i.genProjectFile(ctx, azdCtx, detect)
 	if err != nil {
 		i.console.StopSpinner(ctx, title, input.GetStepResultFormat(err))
 		return err
 	}
 	i.console.Message(ctx, "\n"+output.WithBold("Generating files to run your app on Azure:")+"\n")
 	i.console.StopSpinner(ctx, title, input.StepDone)
 
-	if infraSpec != nil {
-		title = "Generating Infrastructure as Code files in " + output.WithHighLightFormat("./infra")
-		i.console.ShowSpinner(ctx, title, input.Step)
-		err = i.genFromInfra(ctx, azdCtx, *infraSpec)
-		if err != nil {
-			i.console.StopSpinner(ctx, title, input.GetStepResultFormat(err))
-			return err
-		}
-		i.console.StopSpinner(ctx, title, input.StepDone)
-	} else {
-		t, err := scaffold.Load()
-		if err != nil {
-			return fmt.Errorf("loading scaffold templates: %w", err)
-		}
-
-		err = scaffold.Execute(t, "next-steps-alpha.md", nil, filepath.Join(azdCtx.ProjectDirectory(), "next-steps.md"))
-		if err != nil {
-			return err
-		}
-
-		i.console.MessageUxItem(ctx, &ux.DoneMessage{
-			Message: "Generating " + output.WithHighLightFormat("./next-steps.md"),
-		})
-	}
```

Comment:
Keep only the 'else' block contents (`infraSpec` is nil when compose is enabled)